config/routes.rb

```diff
@@ -156,6 +156,7 @@
       patch :flag_for_clearance
     end
 
+    resource :cover_page, only: [:show], path: "cover-page"
```

Comment:
Is there a reason this URL is being overriden with a nicer URL?

Reply:
That's the reason :) 

I hate underscores in URLs